### ISSUE TITLE
credo suggest fails if there's one and only one defexception

### ISSUE THREAD

**Issue by uasi** [rrrene/credo#26]
If target files contain just one `defexception` in total, `mix credo suggest` can detect no functions/modules.

Example:

```ex
# lib/example.ex (let's say no other file is in lib/)

defmodule I_am_snake_case_module_check_me_credo do
end

defmodule TheError do
  defexception [:message]
end

# If uncomment this, credo will start functioning again
#
#defmodule AnotherError do
#  defexception [:message]
#end
```

```
$ mix credo
Checking 1 source file ...

Please report incorrect results: https://github.com/rrrene/credo/issues

Analysis took 0.03 seconds (0.00s to load, 0.03s running checks)
0 mods/funs, found no issues.

Showing priority issues: ↑ ↗ →  (use `--strict` to show all issues, `--help` for options).
```

```
$ uncomment_that_thing
$ mix credo
Checking 1 source file ...

  Code Readability                                                                                                                                                                                          
┃ 
┃ [R] ↗ Module names should be written in PascalCase.
┃       lib/example.ex:1:11 (I_am_snake_case_module_check_me_credo)

(--- snip ---)

Analysis took 0.04 seconds (0.00s to load, 0.04s running checks)
3 mods/funs, found 4 code readability issues.

Showing priority issues: ↑ ↗ →  (use `--strict` to show all issues, `--help` for options).
```

Tested with credo 0.2.4 on on Elixir 1.1.1, Erlang/OTP 18.

Here's clonable working example: https://gitlab.com/uasi/20151223-credo-defexception

**Comment by rrrene** [comment]
Hi! Thx for the bug report. I will fix this as soon as this "x-mas thing" is handled :grin:

**Comment by c-rack** [comment]
We could maybe also warn about defining more than one module per file :smile:

**Comment by rrrene** [comment]
@c-rack we could definitely add a check for that. Please open a separate issue and we can discuss specific points (e.g. there should be an exception for `defexception` modules) :+1:

**Comment by rrrene** [comment]
Just released `v0.2.5` including this. Thx again! :+1:

**Comment by uasi** [comment]
:+1: :100:
